bin/lib/Logos/Generator/Base/Property.pm

```diff
@@ -5,19 +5,33 @@ use Logos::Util;
 sub getterName {
 	my $self = shift;
 	my $property = shift;
-	return Logos::sigil("method", $property->group->name, $property->class->name, $property->getter);
+	return Logos::sigil("method", $property->group->name, $property->class->name, $property->getter, "getter");
```

Comment:
Why this change? The difference for these functions already happens in https://github.com/theos/logos/blob/master/bin/logos.pl#L678-L691

Reply:
This disambiguates it from any methods you might be trying to hook

Reply:
> if you were to %hook a method and try to add a %property with the same name, the generated logos symbols would collide

I'm concerned about people hooking a method and adding a property with the same name (same selector), which I think isn't right.

Reply:
Why isn't it right? It's very common to want to add additional behavior to a getter or setter. There's even an open issue about this, I think it's #41?

Reply:
Undo your additions and replace the first argument `"method"` for `"property"` and we're good.

Reply:
Wow, that is… a much better idea. Dunno why I didn't think to do that instead, lol. Done.